### ISSUE TITLE
Loop index in included nunjucks file

### ISSUE THREAD

**Issue by udayms** [mozilla/nunjucks#258]
This works:

```jinja
{% for item in items %}
    {{ loop.index }}
    {% include "each-item.html" %}
{% endfor %}
```

But, if I move the loop.index into the each-item.html, it doesnt work. Why am I not able to access loop.index inside the included file, while I can access the item object. If I print it once within the loop, then i am able to use it inside the included file.

Am I doing something wrong here?

**Comment by oyyd** [comment]
@carljm This issue seems to have been fixed by #280. Maybe you would like to close it.

**Comment by carljm** [comment]
Yep, thanks for pointing it out!
